Duplicati/Server/KeyManager/PasswordFromDatabase.cs

```diff
@@ -0,0 +1,44 @@
+﻿//  Copyright (C) 2019, The Duplicati Team
+//  http://www.duplicati.com, info@duplicati.com
+//
+//  This library is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as
+//  published by the Free Software Foundation; either version 2.1 of the
+//  License, or (at your option) any later version.
+//
+//  This library is distributed in the hope that it will be useful, but
+//  WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+//  Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public
+//  License along with this library; if not, write to the Free Software
+//  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+using System;
+using Duplicati.Server.Database;
+using Duplicati.Server.Serialization.Interface;
+using System.Linq;
+
+
+namespace Duplicati.Security.KeyManager
+{
+    public class PasswordFromDatabase : IPasswordService
```

Comment:
I don't see any usages of the `PasswordFromDatabase` class or the `IPasswordService` interface in the diff.  Did you intend to reference these somewhere, or are these for future use?

Reply:
This accidentally slipped in due to IDE eagerness of adding unstaged files. I tried to smoothen it out with rebasing but did not manage to do so (due to merge commits or similar), so removed these files in an extra commit.